packages/server-core/src/media/static-resource/static-resource.hooks.ts

```diff
@@ -292,6 +292,36 @@ const addDeleteLog = async (context: HookContext<StaticResourceService>) => {
   }
 }
 
+const addPatchLog = async (context: HookContext<StaticResourceService>) => {
```

Comment:
How is this different from addDeleteLog? Is there a reason the same code can't be reused?

Reply:
only the resource action, I can update the delete log to add the action in the params

Reply:
done c228f29661e37949471c1b41e2b79e38a1b0b18c